.ci_tools/build_samples.sh

```diff
@@ -20,6 +20,7 @@ declare projects=(
     endless-tunnel
     gles3jni
     hello-gl2
+    hello-vulkan
```

Comment:
Please sync to get https://github.com/android/ndk-samples/pull/901 and re-run checks.